README.md

```diff
@@ -249,13 +249,13 @@ SET vectors.k = 32;
 SET LOCAL vectors.k = 32;
 ```
 
+
+### Filter in vector search
+
 If you want to disable vector indexing or prefilter, we also offer some GUC options:
 - `vectors.enable_vector_index`: Enable or disable the vector index. Default value is `on`.
 - `vectors.enable_prefilter`: Enable or disable the prefilter. Default value is `on`.
-
-## Limitations
-
-- The filtering process is not yet optimized. To achieve optimal performance, you may need to manually experiment with different strategies. For example, you can try searching without a vector index or implementing post-filtering techniques like the following query: `select * from (select * from items ORDER BY embedding <-> '[3,2,1]' LIMIT 100 ) where category = 1`. This involves using approximate nearest neighbor (ANN) search to obtain enough results and then applying filtering afterwards.
+More detailed doc at [./docs/filter.md] and issue [#113](https://github.com/tensorchord/pgvecto.rs/issues/113).
 
 ## Setting up the development environment
```

Comment:
It seems that nobody use envd to develop this repo now. Will we afford extra efforts to maintain it?

Reply:
Removed